website/templates/profile/notifications.mako

```diff
@@ -44,13 +45,23 @@
                     <!-- Flashed Messages -->
                     <div data-bind="html: message, attr: {class: messageClass}"></div>
             </div><!--view model scope ends -->
-            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
+                <form id="selectNotifications" class="osf-treebeard-minimal">
+                    <div id="grid"></div>
+                    <div class="help-block" style="padding-left: 15px">
+                            <p id="configureNotificationsMessage"></p>
+                    </div>
+                </form>
```

Comment:
With no save button, the assumption should be that it has saved. If we
can't do a little green checkmark next to the input, then let's go
message-less.

On Thu, Feb 26, 2015 at 4:05 PM, Harry Rybacki <notifications@github.com>
wrote:

> In website/templates/profile/notifications.mako
> <https://github.com/CenterForOpenScience/osf.io/pull/1772#discussion_r25464142>
> :
>
> > @@ -44,13 +45,23 @@
> >                      <!-- Flashed Messages -->
> >                      <div data-bind="html: message, attr: {class: messageClass}"></div>
> >              </div><!--view model scope ends -->
> > -            </div>
> > +        </div>
> > +        <div class="panel panel-default">
> > +            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
> > +                <form id="selectNotifications" class="osf-treebeard-minimal">
> > +                    <div id="grid"></div>
> > +                    <div class="help-block" style="padding-left: 15px">
> > +                            <p id="configureNotificationsMessage"></p>
> > +                    </div>
> > +                </form>
>
> Could we add a growl or something to inform the user that the change has
> been made? I wasn't sure that it had gone through until I manually tested
> the feature after the fact.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/1772/files#r25464142>
> .
>